packages/terra-form/lib/Form.scss

```diff
@@ -0,0 +1,60 @@
+@import '~terra-mixins';
+
+.terra-Form-input,
+.terra-Form-textarea {
+  border: $terra-grey-15 solid 1px;
+  border-radius: 0.25em;
+  font-size: 16px;
+  line-height: 1.428em;
+  padding: 0.178em;
+  width: 100%;
+}
+
+.terra-Form-input {
+  &[type='checkbox'],
+  &[type='radio'] {
+    border: 0;
+    display: inline-block;
+    width: auto;
+  }
+}
+
+.terra-Form-field {
+  margin-bottom: 1.428em;
+}
+
+.terra-Form-control {
+  display: block;
+  margin-bottom: 0.714em;
+  width: auto;
+}
+
+.terra-Form-label {
+  display: block;
+  margin-bottom: 0.357em;
+}
+
+.terra-Form-helpText {
+  display: block;
+  margin-top: 0.357em;
+}
+
+.terra-Form-error {
+  color: $terra-red-70;
+  display: block;
+  margin-top: 0.357em;
+}
+
+@media screen and (min-width: $terra-small-breakpoint) {
+  .terra-Form-field--inline {
+    display: inline-block;
+    margin-right: 0.714em;
```

Comment:
Should we use a mixin here? https://github.com/cerner/terra-core/blob/db2fec58904dd16ccab541dd5d575eb1fc10284e/packages/terra-mixins/src/Mixins.scss#L343